napari/_tests/test_dtypes.py

```diff
@@ -12,7 +12,16 @@
     np.dtype(np.int32),
     np.dtype(np.uint32),
     np.dtype(np.int64),
-    np.dtype(np.uint64),
+    pytest.param(
```

Comment:
Here's a simple suggestion to fix the test IDs that were bothering you - change the decorator on the test below:
```python
@pytest.mark.parametrize('dtype', dtypes, ids=str)
```